inc/Engine/Admin/DomainChange/Subscriber.php

```diff
@@ -57,4 +62,61 @@ public function maybe_launch_domain_changed() {
 		 */
 		do_action( 'rocket_domain_changed', $base_url, $last_base_url );
 	}
+
+	/**
+	 * Check if wp rocket options have been changed.
+	 *
+	 * @return bool
+	 */
+	public function configurations_changed() {
+		if ( ! get_option( self::LAST_OPTION_HASH ) || ! get_option( rocket_get_constant( 'WP_ROCKET_SLUG' ) ) ) {
+			return true;
+		}
+
+		$last_option_hash = get_option( self::LAST_OPTION_HASH );
+
+		$options = get_option( rocket_get_constant( 'WP_ROCKET_SLUG' ) );
+		return rocket_create_options_hash( $options ) !== $last_option_hash;
+	}
+
+	/**
+	 * Save the hash when options are saved.
+	 *
+	 * @param array $oldvalue old options.
+	 * @param array $value new options.
+	 * @return array|void
+	 */
+	public function save_hash_on_update_options( $oldvalue, $value ) {
+		if ( ! is_array( $value ) ) {
+			return;
+		}
+
+		$hash = rocket_create_options_hash( $value );
+
+		update_option( self::LAST_OPTION_HASH, $hash );
+	}
+
+	/**
+	 * Maybe clean cache on domain change.
+	 *
+	 * @return void
+	 */
+	public function maybe_clean_cache_domain_change() {
+		if ( apply_filters( 'rocket_configurations_changed', false ) ) {
```

Comment:
missing docblock for filter

Reply:
@Tabrisrp is there a way to add them to phpcs? I always forget them